Ginger/GingerCoreCommon/Repository/BusinessFlowLib/BusinessFlow.cs

```diff
@@ -45,6 +46,7 @@ public class BusinessFlow : RepositoryItemBase
         private Stopwatch _stopwatch;
         public BusinessFlow()
         {
+            AddCategories();
```

Comment:
as mentioned in other comment, we need better way to initialize it instead of via constructor